_data/navigation.yml

```diff
@@ -27,8 +27,8 @@ primary:
       - text: Getting Started at TTS
         url: getting-started/
         internal: true
-      - text: Onboarding class schedule
-        url: onboarding-schedule/
+      - text: TTS Classes for TTS Onboarding
+        url: tts-classes/
```

Comment:
Not required, but may want to consider a redirect for the old URL

Reply:
Is there an example of that in the handbook I can fork and edit?

Reply:
search for "redirect" in the repo. They're very simple.